store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -49,86 +49,123 @@
+    private static final String H_DESC_VOL_RP         = "Root path";
+    private static final String H_DESC_VOL_STI        = "Store type: internal";
+    private static final String H_DESC_VOL_STE        = "Store type: external";
+    private static final String H_DESC_VOL_STP_S3     = "Storage type: S3";
+    private static final String H_DESC_VOL_STP_OI     = "Storage type: OPENIO";
+    private static final String H_DESC_VOL_CB         = "Compress blobs: true or false";
+    private static final String H_DESC_VOL_CT         = "Compression threshold: default 4KB";
+    private static final String H_DESC_VOL_SMC        = "Non-default store manager class path";
+    private static final String H_DESC_VOL_PRE        = "Volume Prefix";
+    private static final String H_DESC_VOL_BKT_ID     = "S3 Bucket ID";
+    private static final String H_DESC_VOL_AWS_UIT    = "AWS only - Use Intelligent tiering storage class";
+    private static final String H_DESC_VOL_AWS_UFA    = "AWS only - Use Infrequent access storage class";
+    private static final String H_DESC_VOL_AWS_UFAT   = "AWS only - Use Infrequent access storage class blob size threshold";
+    private static final String H_DESC_VOL_URL        = "URL of OpenIO";
+    private static final String H_DESC_VOL_NS         = "Namespace";
+    private static final String H_DESC_VOL_PP         = "Proxy port";
+    private static final String H_DESC_VOL_AP         = "Account port";
+    private static final String H_DESC_VOL_ACC        = "Name of account";
+
+    private static final String H_EXP_INT_VOL_ADD     = "Example: zmvolume -a -t primaryMessage -p /opt/zimbra/sda1 -n internalVolName";
+    private static final String H_EXP_EXT_S3_VOL_ADD  = "Example: zmvolume -a -t primaryMessage -p /opt/zimbra/sda1 -n externalVolName -st external -stp S3 -bid 35d6a471-5218-45f2-b883-62926df94f9b";
+    private static final String H_EXP_EXT_OI_VOL_ADD  = "Example: zmvolume -a -t primaryMessage -p /opt/zimbra/sda1 -n externalVolName -st external -pp 6006 -ap 6000 -acc MY_ACCOUNT -url http://10.10.10.10 -stp OPENIO -ns OPENIO";
+    private static final String H_EXP_INT_VOL_EDIT    = "Example: zmvolume -e -id 20 -n newVolName -t secondaryMessage";
+    private static final String H_EXP_EXT_VOL_EDIT    = "Example: zmvolume -e -id 10 -n newVolName";
+    private static final String H_EXP_VOL_DEL         = "Example: zmvolume -d -id 20";
+
+    private static final String H_NOTE_EXT_OI_VOL_ADD = "Note: Type index volume is not supported for external OPENIO volume";
+    private static final String H_NOTE_EXT_VOL_EDIT   = "Note: Only volume name can be edited for external volumes";
+    private static final String H_STORE_TYPE          = "Store type: internal or external";
+    private static final String H_STORAGE_TYPE        = "Supported storage types are S3 and OPENIO";
+
+    private static final String H_TXT_REQUIRED_ARG    = "<required arg>";
```

Comment:
As discussed remove, if not add any really impacting benefits

Reply:
Done